src/libraries/Common/src/Interop/Unix/Interop.DefaultPathBufferSize.cs

```diff
@@ -0,0 +1,9 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+internal static partial class Interop
+{
+    // Unix max paths are typically 1K or 4K UTF-8 bytes, 256 should handle the majority of paths
+    // without putting too much pressure on the stack.
+    internal const int DefaultPathBufferSize = 256;
```

Comment:
Good question, I don't know if we have data around this. Since this number is used in some other APIs (and this const is now consumed in those places) I would rather not make that change now. Feel free to open an issue though.